attributes/default.rb

```diff
@@ -359,6 +359,14 @@
 default['datadog']['process_agent']['container_interval'] = nil
 default['datadog']['process_agent']['rtcontainer_interval'] = nil
 
+# System probe functionality settings
+default['datadog']['system_probe']['enabled'] = false
+# sysprobe_socket defines the unix socket location
+default['datadog']['system_probe']['sysprobe_socket'] = '/opt/datadog-agent/run/sysprobe.sock'
+# debug_port is the http port for expvar, it is disabled if not specified
+default['datadog']['system_probe']['debug_port'] = nil
```

Comment:
0?

Reply:
nil and 0 are the same, right?

Reply:
I'm not sure how ruby interprets them. What gets injected into the config?

Reply:
It's just going to put `0` in the config file. What I'm saying is port 0 is the same in config as empty. To me using empty makes more sense than `0`.

Reply:
Can we be explicit and leave the default as 0 here? I have no idea where the conversion from nil to zero happens.